enginetest/queries/transaction_queries.go

```diff
@@ -1064,4 +1064,56 @@ var TransactionTests = []TransactionTest{
 			},
 		},
 	},
+	{
+		Name: "ddl queries are implicitly committed",
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "/* client a */ set @@autocommit = 0;",
+				Expected: []sql.Row{{}},
+			},
+			{
+				Query:    "/* client a */ start transaction;",
+				Expected: []sql.Row{},
+			},
+			{
+				// This implicitly commits the transaction
+				Query:    "/* client a */ create table t (pk int primary key);",
+				Expected: []sql.Row{{types.OkResult{}}},
+			},
+			{
+				Query:    "/* client a */ commit;",
```

Comment:
Remove this right? The whole point is to check for an implicit commit, not an explicit one

Reply:
Yep, removed